src/main/java/it/pagopa/interop/probing/eservice/operations/annotations/ValidateStringArraySize.java

```diff
@@ -0,0 +1,23 @@
+package it.pagopa.interop.probing.eservice.operations.annotations;
+
+import it.pagopa.interop.probing.eservice.operations.annotations.validator.StringArrayValidator;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Target({ ElementType.FIELD })
+@Retention(RetentionPolicy.RUNTIME)
+@Constraint(validatedBy = StringArrayValidator.class)
+public @interface ValidateStringArraySize {
+
+	int maxSize();
+
+	String message() default "One of the strings of the array is more than {maxSize} characters long";
```

Comment:
We can start to interpolate message with i18n for internationalization

Reply:
This is a generic topic to eventually apply to all repositories. Maybe it would be good to discuss it in a dedicated call, we have internationalization on the frontend, we can discuss if to use it in backend too